pkgbuild/packagesbuild.sh

```diff
@@ -65,42 +98,36 @@ while test $# -gt 0; do
   esac
 done
 
-rm -rf *.pkg distribution.xml Resources/en.lproj/welcome.html Resources/en.lproj/conclusion.html
 mkdir -p "${INPUT_DIRECTORY}/Contents/Home/bundle/Libraries"
 if [ -f "${INPUT_DIRECTORY}/Contents/Home/lib/server/libjvm.dylib" ]; then
   cp "${INPUT_DIRECTORY}/Contents/Home/lib/server/libjvm.dylib" "${INPUT_DIRECTORY}/Contents/Home/bundle/Libraries/libserver.dylib"
 else
   cp "${INPUT_DIRECTORY}/Contents/Home/jre/lib/server/libjvm.dylib" "${INPUT_DIRECTORY}/Contents/Home/bundle/Libraries/libserver.dylib"
 fi
-# Detect if JRE or JDK
-case $INPUT_DIRECTORY in
-  *-jre)
-    TYPE="jre"
+
+if [ $TYPE == "jre" ]; then
     /usr/libexec/PlistBuddy -c "Add :JavaVM:JVMCapabilities array" "${INPUT_DIRECTORY}/Contents/Info.plist"
     /usr/libexec/PlistBuddy -c "Add :JavaVM:JVMCapabilities:0 string CommandLine" "${INPUT_DIRECTORY}/Contents/Info.plist"
-    ;;
-  *)
-    TYPE="jdk"
-    ;;
-esac
+fi
 
-# Plist commands:
 case $JVM in
   openj9)
-    IDENTIFIER="net.adoptopenjdk.${MAJOR_VERSION}-openj9.${TYPE}"
-    DIRECTORY="adoptopenjdk-${MAJOR_VERSION}-openj9.${TYPE}"
-    BUNDLE="AdoptOpenJDK (OpenJ9)"
+    IDENTIFIER="net.${VENDOR}.${MAJOR_VERSION}-openj9.${TYPE}"
+    DIRECTORY="${VENDOR}-${MAJOR_VERSION}-openj9.${TYPE}"
+    BUNDLE="${PACKAGE_NAME} (OpenJ9)"
+    cp Licenses/license-OpenJ9.en-us.rtf Resources/license.rtf
     case $TYPE in
-      jre) BUNDLE="AdoptOpenJDK (OpenJ9, JRE)" ;;
-      jdk) BUNDLE="AdoptOpenJDK (OpenJ9)" ;;
+      jre) BUNDLE="${PACKAGE_NAME} (OpenJ9, JRE)" ;;
+      jdk) BUNDLE="${PACKAGE_NAME} (OpenJ9)" ;;
     esac
     ;;
   *)
-    IDENTIFIER="net.adoptopenjdk.${MAJOR_VERSION}.${TYPE}"
-    DIRECTORY="adoptopenjdk-${MAJOR_VERSION}.${TYPE}"
+    IDENTIFIER="net.${VENDOR}.${MAJOR_VERSION}.${TYPE}"
```

Comment:
Ideally, it should be possible to redeclare the entire identifier.

Reply:
add `--identifier`